setup-local/src/binaryControl.js

```diff
@@ -0,0 +1,232 @@
+const tc = require('@actions/tool-cache');
+const io = require('@actions/io');
+const exec = require('@actions/exec');
+const core = require('@actions/core');
+const github = require('@actions/github');
+const os = require('os');
+const path = require('path');
+const fs = require('fs');
+const Utils = require('./utils');
+const ArtifactsManager = require('./artifacts');
+const constants = require('../config/constants');
+
+const {
+  BINARY_LINKS,
+  LOCAL_BINARY_FOLDER,
+  PLATFORMS,
+  LOCAL_BINARY_NAME,
+  LOCAL_LOG_FILE_PREFIX,
+  LOCAL_BINARY_TRIGGER,
+  ALLOWED_INPUT_VALUES: {
+    LOCAL_TESTING,
+  },
+  ENV_VARS: {
+    BROWSERSTACK_LOCAL_LOGS_FILE,
+  },
+} = constants;
```

Comment:
Did we just include only the required constants? 😮

Reply:
not really. we `required` the whole file first and de-structured the required ones.

Reply:
resolving this. no issue here.